doc/makefile-variables.txt

```diff
@@ -107,3 +107,25 @@ that need to be applied to the test output for just this component.
   have been found which have required altering this to "-xO3".  There are also
   studio_OPT.$(MACH).$(BITS) versions of this available if greater specificity
   is needed.
+
+If you frequently rebuild the same code, such as when you maintain a build server
+or iterate recipes for the same component, you can benefit by caching the build
+products with "ccache" - so for repeated input conditions you'd get same output
+objects quickly.
+
+* CCACHE_DISABLE is an inverted toggle of the "ccache" program, which our common
+  shared-macros.mk Makefile also honours. By original project's intent, if you
+  execute "ccache" then you want it to do its job. If you are debugging it or
+  unsure if it causes problems, you can easily temporarily disable the "ccache"
```

Comment:
I'd rather not add the re-inverted variable, because 1) in the future a custom `CCACHE_*` variable might conflict with what the original project offers, and 2) we'd have two different toggles for the same feature whose settings can become conflicting and not apparent to users.